### PR TITLE
fix hypershift-extended-enable-qe-catalogsource step failed with disconnected

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-baremetalds-hypershift-agent-mce-mgmt-f7

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-4.18-upgrade-from-stable-4.17-aws-ipi-disc-priv-f28

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-baremetalds-hypershift-agent-mce-mgmt-f7

**Comment by LiangquanLi930** [comment]
hypershift job looks good
```
INFO[2024-09-19T07:40:32Z] Running step baremetalds-hypershift-agent-mce-mgmt-f7-cucushift-hypershift-extended-enable-qe-catalogsource. 
INFO[2024-09-19T07:41:23Z] Step baremetalds-hypershift-agent-mce-mgmt-f7-cucushift-hypershift-extended-enable-qe-catalogsource succeeded after 51s. 
INFO[2024-09-19T07:41:23Z] Running step baremetalds-hypershift-agent-mce-mgmt-f7-hypershift-agent-create-metallb. 
INFO[2024-09-19T07:50:07Z] Step baremetalds-hypershift-agent-mce-mgmt-f7-hypershift-agent-create-metallb succeeded after 8m43s. 
```

**Comment by LiangquanLi930** [comment]
/pj-rehearse pull-ci-openshift-origin-release-4.16-e2e-agent-disconnected-ovn-dualstack-metal3

File: ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh [hunk 1]
```diff
@@ -519,7 +519,7 @@ set_CA_for_nodes
 #
 #ocp_version=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '.' -f1,2)
 kube_major=$(oc version -o json |jq -r '.serverVersion.major')
-kube_minor=$(oc version -o json |jq -r '.serverVersion.minor' |sed 's/[^0-9\.]//g')
+kube_minor=$(oc version -o json |jq -r '.serverVersion.minor' | sed 's/+$//')
```
Comment (jianzhangbjz) [inline ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh:522]: Seems like the `sed 's/[^0-9\.]//g'` is better.

**Comment by jianzhangbjz** [comment]
Test pass: https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/56900/rehearse-56900-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-baremetalds-hypershift-agent-mce-mgmt-f7/1836647662938492928/artifacts/baremetalds-hypershift-agent-mce-mgmt-f7/cucushift-hypershift-extended-enable-qe-catalogsource/build-log.txt
/lgtm
/approve

**Pull request merged**